config/locales/pt-BR.yml

```diff
@@ -0,0 +1,175 @@
+pt-BR:
+  title: "RubyGems.org"
+  subtitle: "o host de gems da sua comunidade"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Accesso Negado. Por favor se registre em: https://rubygems.org"
+  feed_subscribed: "RubyGems.org | Gems do seu Feed"
+  feed_latest: "RubyGems.org | Últimas Gems"
+  time_ago: "%{duration} atrás"
+  form_disable_with: "Atualizando..."
+  update: "Atualizar"
+  edit: "Editar"
+  sign_in: "Logar"
+  sign_up: "Cadastrar"
+  footer_about: "RubyGems.org é o serviço de hospedagem da comunidade Ruby. Publique e instale suas gems instantaneamente. Use a API para interagir e encontrar mais informações sobre gems disponíveis. Torne-se um contribuidor e melhore o site com suas mudanças."
+  locale_name: "Português do Brasil"
+  this_rubygem_could_not_be_found: "A rubygem não foi encontrada."
+
+  dashboards:
+    show:
+      latest: "Últimas Atualizações"
+      latest_title: "Últimas Atualizações | Feed RSS"
+      no_subscriptions: "Você ainda não está observando nenhuma gem. Visite uma %{gem_link} para poder observá-la!"
+      gem_link_text: "Página da gem"
+      mine: "Minhas Gems"
+      no_owned: "Você ainda não enviou nenhuma gem. Verifique os guias sobre %{creating_link} uma gem ou %{migrating_link} uma gem do RubyForge."
+      creating_link_text: 'criar'
+      migrating_link_text: 'migrar'
+      my_subscriptions: 'Observando'
+
+  home:
+    index:
+      downloads_counting: 'downloads'
+      find_blurb: "Encontre, instale e publique RubyGems."
+      learn:
+        install_rubygems: "Instalar o RubyGems"
+
+  layouts:
+    application:
+      header:
+        dashboard: "Painel de Controle"
+        sign_in: "Logar"
+        sign_up: "Cadastrar"
+        sign_out: "Deslogar"
+        search_gem: "Buscar Gems&hellip;"
+      footer:
+        status: "Status"
+        uptime: "Uptime"
+        source_code: "Código Fonte"
+        data: "Dump de Dados"
+        discussion_forum: "Fóruns"
+        statistics: "Estatísticas"
+        blog: "Blog"
+        about: "Sobre o Rubygems"
+        help: "Ajuda"
+        api: "API"
+        guides: "Guias"
+        contribute: "Como Contribuir"
+        supported_by: "Patrocínio"
+        hosted_by: "Hospedagem"
+        designed_by: "Design"
+        resolved_with: "DNS"
+        optimized_by: "Otimizações"
+        tracking_by: "Coleta de Dados"
+        monitored_by: "Monitoramento"
+        gems_served_by: "Provisionamento"
+
+  pages:
+    about:
+      title: Sobre o Rubygems
+      purpose:
+        header: "Bem vindo ao %{site}, a comunidade do serviço de hospedagem de gems. Este projeto tem três propósitos:"
```

Comment:
Like you had for the `footer_about` above, it could be `o serviço de hospedagem de gems da comunidade Ruby`.